package.json

```diff
@@ -9,8 +9,8 @@
     "dev": "babel -w src -d lib",
     "lint": "eslint src test",
     "fix": "eslint src test --fix",
-    "test": "mocha -t 5000",
-    "ci": "npm run lint && cross-env DEBUG_CALLS=true npm run test",
+    "test": "mocha -t 20000",
+    "ci": "npm run lint && cross-env npm run test",
```

Comment:
Seems fine to me.  We removed it in Asini, too.

We could actually also remove the `cross-env`, too, since that was just for getting the environment variable set consistently across environments.  Then we could drop the dep, I think.

@hzoo @evocateur @doug-wade Any objection to dropping this?